src/test/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The "+ getName() + " began its work.");
```

Comment:
The message in the 'doWork' method is not very informative. According to the checklist, you should indicate what type of Machine is working. Consider specifying that an Excavator is working in the message.